<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menugen</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="labelRecipes">
        <property name="text">
         <string>Recettes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="RecipesView"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="MenuView">
        <property name="text">
         <string>Menus</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>758</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <addaction name="actionNewRecipe"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Editer">
    <property name="title">
     <string>&amp;Editer</string>
    </property>
    <addaction name="actionGenMenu"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Editer"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
   <property name="shortcut">
    <string extracomment="shortcut to quit the app">Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNewRecipe">
   <property name="text">
    <string>&amp;Nouvelle Recette</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionGenMenu">
   <property name="text">
    <string>&amp;Générer un menu</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
